evm/scripts/merge.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python3
+
+import csv
+import os
+
+RESULT_CSV_OUTPUT_PATH = "/evmraceresults/"
+EVMS = ["evmone", "parity", "geth", "cita-vm"]
+RESULT_FILE = os.path.join(RESULT_CSV_OUTPUT_PATH + "evm_benchmarks.csv")
+
+def main():
+    fieldnames = ['engine', 'test_name', 'total_time', 'gas_used']
```

Comment:
gas used  for evmone doesn't use same format as other clients.  e.g. `1.5K` for evmone vs `1500` for other clients.